packages/app-builder-lib/src/util/yarn.ts

```diff
@@ -12,7 +11,7 @@ import { PM, detect, getPackageManagerVersion } from "../node-module-collector"
 import { NodeModuleDirInfo } from "./packageDependencies"
 import { rebuild as remoteRebuild } from "./rebuild/rebuild"
 
-export async function installOrRebuild(config: Configuration, appDir: string, options: RebuildOptions, forceInstall = false) {
+export async function installOrRebuild(config: Configuration, appDir: string, projectDir: string, options: RebuildOptions, forceInstall = false) {
```

Comment:
For **all** instances of where we have `appDir: string, projectDir: string`, can we structure it as an object? Having 2 string params makes it easy to get swapped by another dev in the future.
```
installOrRebuild(config: Configuration, dirs: { appDir: string, projectDir: string }, options: RebuildOptions, forceInstall = false)
```
`{ appDir: string, projectDir: string }` could be extracted to a type somewhere too

Reply:
Good suggestion.

Reply:
Fixed.